views/forms/ControlledReactSelect/components.js

```diff
@@ -26,55 +23,36 @@ export const components = {
       key={props.children}
       buttonRef={props.innerRef}
       selected={props.isFocused}
-      className={
-        props.isSelected
-          ? props.selectProps.classes.selectedFontWeight
-          : props.selectProps.classes.normalFontWeight
-      }
       {...props.innerProps}
     >
       {props.children}
     </MenuItem>
   ),
   NoOptionsMessage: (props) => (
-    <Typography
-      color="textSecondary"
-      className={props.selectProps.classes.noOptionsMessage}
-      {...props.innerProps}
-    >
+    <Typography color="textSecondary" {...props.innerProps}>
       {props.children}
     </Typography>
   ),
   Control: (props) => (
     <TextField
       fullWidth
-      InputProps={{
-        inputComponent,
-        inputProps: {
-          className: props.selectProps.classes.input,
-          inputRef: props.innerRef,
-          children: props.children,
-          ...props.innerProps,
-        },
-      }}
+      // InputProps={{
+      //   inputComponent,
+      //   inputProps: {
+      //     inputRef: props.innerRef,
+      //     children: props.children,
+      //     ...props.innerProps,
+      //   },
+      // }}
```

Comment:
can we remove this commented out code?

Reply:
[6b310d1](https://github.com/AMP-SCZ/dpdash/pull/458/commits/6b310d18b25129974d8ce470610170164ed5d426)

Reply:
@Elkrival this still needs to be removed.